index.js

```diff
@@ -36,12 +36,11 @@ load.path = function (dir) {
     if (debug) return debug
   }
 
-  // Find most specific flavor first
-  var prebuilds = path.join(dir, 'prebuilds', platform + '-' + arch)
-  var parsed = readdirSync(prebuilds).map(parseTags)
-  var candidates = parsed.filter(matchTags(runtime, abi))
-  var winner = candidates.sort(compareTags(runtime))[0]
-  if (winner) return path.join(prebuilds, winner.file)
+  var prebuild = resolve(dir)
+  if (prebuild) return prebuild
+
+  var nearby = resolve(path.dirname(process.execPath))
+  if (nearby) return nearby
```

Comment:
What happens if you have multiple addons using `node-gyp-build`? Won't they conflict?

Reply:
And what happens in a packed Electron app (e.g. with [`electron-builder`](https://github.com/electron-userland/electron-builder))? Which doesn't have a "real" execPath IIRC.

Reply:
@emilbayes

Reply:
I don’t think this is relevant for electron but yes we should namespace the location here. This was a quick fix to work around a cli bundling issue

Reply:
@mafintosh Is there a relevant GH issue I can read to learn more?

And re: electron, I'm a bit worried that this will accidentally load e.g. `/tmp/foo.node`.

Reply:
No issue sorry, we were debugging together. Can you explain the last part? About loading foo.node

Reply:
I vaguely recall that `process.execPath` is a temporary location in some cases, but I can't find documentation on that just now. Maybe I'll try to test it later this week.